packages/manager/modules/log-to-customer/src/translations/Messages_fr_FR.json

```
@@ -1,9 +1,16 @@
 {
   "log_tab_title": "Logs",
   "log_tile_section_title": "Comment utiliser vos logs",
+  "log_tile_subscriptions_title": "Utiliser vos logs",
   "log_tile_title": "Flux",
   "log_tile_desc_create_account": "Pour activer ce transfert vous devez d’abord créer un compte Logs Data Platform et un premier flux de données.",
   "log_tile_desc_create_stream": "Pour activer ce transfert vous devez d’abord créer un premier flux de données dans votre compte Logs Data Platform.",
+  "log_title_subscriptions_title": "Ces logs sont transférés dans Logs Data Platform",
+  "log_title_subscriptions_user": "Utilisateur",
+  "log_title_subscriptions_stream": "Flux de données",
+  "log_title_subscriptions_button_watch": "Observer sur graylog",
+  "log_title_subscriptions_unsubscribe": "Arrêter le transfert",
+  "log_tile_subscriptions_subscribe": "Souscrire sur un autre compte",
```

Sounds weird in french, either `pour` or `avec` sounds better
```suggestion
  "log_tile_subscriptions_subscribe": "Souscrire (pour|avec) un autre compte",
```